### ISSUE TITLE
Please add blocked user name to blocked messages

### ISSUE THREAD

**Issue by urda** [LemmyNet/lemmy#3786]
### Requirements

- [X] Is this a feature request? For questions or discussions use https://lemmy.ml/c/lemmy_support
- [X] Did you check to see if this issue already exists?
- [X] Is this only a feature request? Do not put multiple feature requests in one issue.
- [X] Is this a backend issue? Use the [lemmy-ui](https://github.com/LemmyNet/lemmy-ui) repo for UI / frontend issues.

### Is your proposal related to a problem?

Much like #3698 ...

We should also note the user name in these blocked messages:

https://github.com/LemmyNet/lemmy/blob/963d04b3526f8a5e9ff762960bfb5215e353bb27/crates/apub/src/activities/mod.rs#L71-L93

### Describe the solution you'd like.

The log message should be updated, like #3698 to include the blocked user names.

### Describe alternatives you've considered.

Not many, best to have it in log.

**Comment by urda** [comment]
@Freakazoid182 ya think you could help here too?

**Comment by Freakazoid182** [comment]
@urda Seems like a the same change, so happy to do it. 
I would note that putting user names in logs might be putting PII data in places were we wouldn't want it.
I'm not totally sure if that's an issue here or not. I can make a pull request in any case, and see what the devs think.

**Comment by urda** [comment]
Hell yeah. Awesome. And yeah good call with the PII but let the devs reject it if so.

I don't think it is an issue, as logs are for the operators.

**Comment by urda** [comment]
My hero @Freakazoid182 !

Closed via #3855

**Comment by urda** [comment]
I'm not sure if this is "fixed". In `0.19.3` all the messages just say:

```
WARN lemmy_server::root_span_builder: PersonIsBannedFromSite: PersonIsBannedFromSite 
```

Looks like it's a string literal instead of the username.

**Comment by Nutomic** [comment]
Not sure what you are looking at, the code definitely sets the username:

https://github.com/LemmyNet/lemmy/blob/43f20881cb2057d5784fac69a8944c00e57d5042/crates/apub/src/activities/mod.rs#L84

https://github.com/LemmyNet/lemmy/blob/43f20881cb2057d5784fac69a8944c00e57d5042/crates/apub/src/objects/private_message.rs#L118

**Comment by urda** [comment]
@Nutomic in `0.19.3` this is what the log reports to me, without the username:

```
2024-09-28T20:53:33.564112Z  WARN lemmy_server::root_span_builder: PersonIsBannedFromSite: PersonIsBannedFromSite
   0: lemmy_apub::activities::verify_person_in_community
             at crates/apub/src/activities/mod.rs:74
   1: lemmy_apub::activities::create_or_update::comment::verify
             at crates/apub/src/activities/create_or_update/comment.rs:116
   2: lemmy_apub::activities::community::announce::receive
             at crates/apub/src/activities/community/announce.rs:153
   3: lemmy_server::root_span_builder::HTTP request
           with http.method=POST http.scheme="https" http.host=lebowski.social http.target=/inbox otel.kind="server" request_id=1e1a4548-c657-429b-92b3-4bd25a8538b2
             at src/root_span_builder.rs:16
```

the

```
PersonIsBannedFromSite: PersonIsBannedFromSite
```

does not include the information I would expect here.

**Comment by Nutomic** [comment]
The `root_span_builder` is gone in the main branch, and the username also gets included. So this will be fixed in 0.20.
